components/Collections/src/MDCCollectionViewController.m

```diff
@@ -434,12 +434,12 @@ - (BOOL)collectionView:(UICollectionView *)collectionView
 
 - (void)collectionView:(UICollectionView *)collectionView
```

Comment:
We already do require calling `super` on some of the methods that show/hide the infoBar.  You can see this here: https://github.com/material-components/material-components-ios/blob/develop/components/Collections/src/MDCCollectionViewController.h#L47

Is there anything further you think might be required here?